packages/search-engine/src/direflow-components/search-engine/components/SearchField.tsx

```diff
@@ -102,7 +102,18 @@ const SearchField: React.FunctionComponent = () => {
     
 
     const partyCheckBox = (
-        <Box sx={{ display: 'flex', flexDirection: 'row', ml: 3 }}>
+        <Box sx={{ 
+            display: 'flex', 
+            flexDirection: 'column', 
+            padding: '3px',
+            paddingLeft: '10px', 
+            border: 3,
+            borderRadius: 2, 
+            borderColor: `${alpha('#1976d2', 0.75)}` 
```

Comment:
same thing here, is it different if we use the `rgba` value?
`bgcolor: 'rgba(10, 46, 82, 0.75)'`